openshift/am-pattern-2/README.md

```diff
@@ -0,0 +1,73 @@
+# Openshift resources of WSO2 API Manager with WSO2 API Manager Analytics
+
+![WSO2 API Manager pattern 2 deployment](pattern-2.png)
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+
+## Prerequisites
+
+1. An already setup OpenShift cluster
+
+2. Seven NFS shares for storage. For a step by step guide on how to install NFS server refer here. [CentOS](https://www.server-world.info/en/note?os=CentOS_7&p=nfs&f=1), [Ubuntu](https://www.server-world.info/en/note?os=Ubuntu_18.04&p=nfs&f=1)
+
+## Quick Start Guide
+
+1. Create an OpenShift project name `wso2`.
+
+	`oc new-project wso2`
+
+2. Add your subscription details
+
+	`oc create secret docker-registry wso2-image-pull-secret --docker-server=docker.wso2.com --docker-username=<SUBSCRIPTION_USERNAME> --docker-password=<SUBSCRIPTION_PASSWORD>`
+
+3. Deploy WSO2 APIM Server using the Template.yaml
+
+	`oc process -f Template.yaml -p NFS_SERVER_IP=<NFS_SERVER_IP> -p NFS_SHARE_DATABASE=<DATABASE_NFS_SHARE_PATH> -p NFS_SHARE_SYNAPSE=<SYNAPSE_NFS_SHARE_PATH> -p NFS_SHARE_EXECUTION_PLANS=<EXECUTION_PLANS_SHARE_PATH> -p NFS_SHARE_CARBON_DB_1=<CARBON_DB_1_NFS_SHARE_PATH> -p NFS_SHARE_SOLR_1=<SOLR_1_NFS_SHARE_PATH> -p NFS_SHARE_CARBON_DB_2=<CARBON_DB_2_NFS_SHARE_PATH> -p NFS_SHARE_SOLR_2=<SOLR_2_NFS_SHARE_PATH> -p OPENSHIFT_BASE_DOMAIN=<OPENSHIFT_URL> | oc create -f -`
+
+4. Access product management consoles. Obtain the (HOST/PORT) of the route resource
+
+	`oc get route -n <NAMESPACE>`
```

Comment:
Is it a namespace or project?

Reply:
Fix in https://github.com/wso2/openshift-apim/pull/1/commits/37ccb91b0e914a3c7390a4a610df08202b3226f9 commit. Change namespace to project as its the terminology used in openshift